docs/changelog.md

```diff
@@ -12,6 +12,9 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Config.kt)
 
+* **browser-engine-system**
+  * ⚠️ **This is a breaking API change**: built-in `WebView`'s on-screen zoom controls are hidden by default.
```

Comment:
🤔 @dector @pocmo is this really a breaking change? To me, it looks more like a underneath behavior change.

Reply:
Yeah, it's more like a behavior change.

Reply:
`SystemEngineSession` is public class. It's behavior-by-default was changed. So I've added label "This is a breaking API change". Should I change it to "This is a breaking behavior change" or remove completely?

Reply:
I think  "This is a breaking behavior change" is more suitable.

Reply:
:+1: I agree. One moment.